tests/integration/targets/test_create_rds_global_cluster/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: Integration tests for 'create_rds_global_cluster' role
+  module_defaults:
+    group/amazon.cloud.aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+
+  # Due to CI limitations, it is not possible to create a replica in another region
+  # and with AWS limitations we cannot create a replica cluster in same region as primary cluster i.e. us-east-1
+
+  # block:
+  #   - name: create global db
+  #     ansible.builtin.include_role:
+  #       name: cloud.aws_ops.create_rds_global_cluster
+  #     vars:
+  #       create_rds_global_cluster_operation: create
+  #       create_rds_global_cluster_enigne: "{{ test_enigne }}"
+  #       create_rds_global_cluster_engine_version: "{{ test_engine_version }}"
+  #       create_rds_global_cluster_instance_class: "{{ test_instance_class }}"
+  #       create_rds_global_cluster_username: "{{ test_username }}"
+  #       create_rds_global_cluster_password: "{{ test_password }}"
+  #       create_rds_global_cluster_global_cluster_name: "{{ test_global_cluster_name }}"
+  #       create_rds_global_cluster_primary_cluster_name: "{{ test_primary_cluster_name }}"
+  #       create_rds_global_cluster_primary_cluster_region: "{{ test_primary_cluster_region }}"
+  #       create_rds_global_cluster_primary_cluster_instance_name: "{{ test_primary_cluster_instance_name }}"
+  #       create_rds_global_cluster_replica_cluster_name: "{{ test_replica_cluster_name }}"
+  #       create_rds_global_cluster_replica_cluster_region: "{{ test_replica_cluster_region }}"
+  #       create_rds_global_cluster_replica_cluster_instance_name: "{{ test_replica_cluster_instance_name }}"
+  #       create_rds_global_cluster_replica_enable_global_write_forwarding: "{{ test_replica_enable_global_write_forwarding }}"
+
+  #   - name: Get Global DB information
+  #     amazon.aws.rds_cluster_info:
+  #       cluster_id: "{{ test_global_cluster_name }}"
```

Comment:
I guess you should use the `amazon.cloud.rds_global_cluster` module here instead.